packages/kube-client/lib/resources/GardenerCore.js

```diff
@@ -62,6 +62,10 @@ class SecretBinding extends mix(GardenerCore).with(NamespaceScoped, Readable, Ob
 }
 
 class Shoot extends mix(GardenerCore).with(NamespaceScoped, Readable, Observable, Writable) {
+  adminKubeconfig (namespace, name, body, options) {
```

Comment:
Rename it to `requestAdminKubeconfig` or may be better `createAdminKubeconfigRequest`. This will make a POST request and it should not look like a GET method.

Reply:
✅